staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/defaulting/validation.go

```diff
@@ -86,7 +86,11 @@ func validate(pth *field.Path, s *structuralschema.Structural, rootSchema *struc
 			} else if errs := apiservervalidation.ValidateCustomResource(pth.Child("default"), s.Default.Object, validator); len(errs) > 0 {
 				allErrs = append(allErrs, errs...)
 			} else if celValidator := cel.NewValidator(s); celValidator != nil {
-				allErrs = append(allErrs, celValidator.Validate(pth.Child("default"), s, s.Default.Object)...)
+				err, remainingBudget := celValidator.Validate(pth.Child("default"), s, s.Default.Object, cel.RuntimeCELCostBudget)
```

Comment:
RuntimeCELCostBudget is supposed to be object-wide, but this is called multiple times in the course of validating a single CRD with multiple defaults, right? should it be set at the top level and decremented as multiple defaults are validated? same comment below

Reply:
I will move it upper together with the const in the following commit